package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "memcache-plus",
+  "name": "@feedfm/memcache-plus",
```

Comment:
Please remove this change

Reply:
Looks like this PR is a request to merge from a fork. In the interest of time, I'll submit a new PR off a branch instead, with an accompanying Issue, so we can get this done today.

Reply:
Crap yeah I see that, I'll do it then, hang tight!

Reply:
All done - see new PR #95